pywps/app/Process.py

```diff
@@ -172,20 +173,32 @@ def _execute_process(self, async, wps_request, wps_response):
             if running >= maxparallel and maxparallel != -1:
                 raise ServerBusy('Maximum number of parallel running processes reached. Please try later.')
             wps_response._update_status(WPS_STATUS.ACCEPTED, u"PyWPS Request accepted", 0)
-            wps_response = self._run_process(wps_request, wps_response)
+            wps_response = self._run_sync(wps_request, wps_response)
```

Comment:
why is it called `run_sync`? docker will only be run in async mode.

Reply:
We discussed that with @jachym and we concluded that the docker should be run even in sync mode once the ``mode`` parameter is set to ``docker`` in config file.